src/Util/ColumnParser.php

```diff
@@ -191,7 +190,6 @@ public function getType($field, $type)
         $collection = new Collection($reflector->getConstants());
 
         $validTypes = $collection->filter(function ($value, $constant) {
-            $value;
```

Comment:
Atom has integration with all manner of linting programs, and in this case it was complaining about unused variables.